docs/supported_code_analyzers.md

```diff
@@ -0,0 +1,30 @@
+## Code Analysis tools
+| Language       | Analyzer     | Supported by CodeChecker |
+| -------------- |--------------|---------------------|
+| **C/C++**      | [Clang Tidy](https://clang.llvm.org/extra/clang-tidy/)  | ✓ |
+|                | [Clang Static Analyzer](https://clang-analyzer.llvm.org/)    | ✓ |
+|                | [Clang Sanitizers](#clang-sanitizers)    | ✗ |
+|                | [Cppcheck](http://cppcheck.sourceforge.net/)    | ✓ |
+|                | [Infer](https://fbinfer.com/)    | ✗ |
+| **Java**       | [FindBugs](http://findbugs.sourceforge.net/)    | ✗ |
```

Comment:
The successor of FindBugs is [Spotbugs](https://spotbugs.github.io/) I think we should list that too.